apps/dashboard/src/components/contract-components/published-contract/index.tsx

```diff
@@ -235,23 +234,6 @@ export const PublishedContract: React.FC<PublishedContractProps> = ({
                             : "Proxy"}{" "}
                           Enabled
                         </Heading>
-                        {implementationAddresses &&
```

Comment:
Internal discussion:
https://thirdwebdev.slack.com/archives/C04PAQYN5HB/p1727426894785319

Reply:
@jnsdls  Can you confirm this as welll?

Reply:
Should be fine, as Joaquim said we need to re-do it later but it’ll be a lot simpler UI wise